lib/util/abi_crosswalk.json

```diff
@@ -2598,5 +2662,57 @@
   "17.1.0": {
     "node_abi": 102,
     "v8": "9.5"
+  },
+  "17.2.0": {
+    "node_abi": 102,
+    "v8": "9.6"
+  },
+  "17.3.0": {
+    "node_abi": 102,
+    "v8": "9.6"
+  },
+  "17.3.1": {
+    "node_abi": 102,
+    "v8": "9.6"
+  },
+  "17.4.0": {
+    "node_abi": 102,
+    "v8": "9.6"
+  },
+  "17.5.0": {
+    "node_abi": 102,
+    "v8": "9.6"
+  },
+  "17.6.0": {
+    "node_abi": 102,
+    "v8": "9.6"
+  },
+  "17.7.0": {
+    "node_abi": 102,
+    "v8": "9.6"
+  },
+  "17.7.1": {
+    "node_abi": 102,
+    "v8": "9.6"
+  },
+  "17.7.2": {
+    "node_abi": 102,
+    "v8": "9.6"
+  },
+  "17.8.0": {
+    "node_abi": 102,
+    "v8": "9.6"
+  },
+  "17.9.0": {
+    "node_abi": 102,
+    "v8": "9.6"
+  },
+  "18.0.0": {
+    "node_abi": 108,
+    "v8": "10.1"
+  },
+  "18.1.0": {
+    "node_abi": 108,
+    "v8": "10.1"
   }
```

Comment:
Below a ⛔ symbol appears because the GitHub editor wants all text files to end with one and only one blank line.